src/sagemaker_mxnet_serving_container/utils.py

```diff
@@ -58,3 +58,18 @@ def read_data_shapes(path, preferred_batch_size=1):
         data_shapes.append((name, shape))
 
     return data_names, data_shapes
+
+
+# TODO (@bvveeram): This function is also implemented in the
+# sagemaker-inference package. Once the MXNet 1.4 EIA image is deprecated, pin
+# the sagemaker-inference dependency to >= 1.5.0, remove this function, and use
+# the equivalent function in sagemaker-inference.
```

Comment:
Would you be able to add an issue to the repo after this merge? Thanks!

Reply:
Yes, will do.